### ISSUE TITLE
Avoid layout jumps when loading a page

### ISSUE THREAD

**Issue by kai687** [kai687/sphinxawesome-theme#1718]
Something about the sidebar and hiding nested sections. Whenever you load a new page, you get this weird little twerk of hiding the nested section when the JavaScript is loaded. Figure out a way to avoid that.

**Comment by dylanh724** [comment]
Yep, if I switch to dark mode and nav to a new page, I'll see a flash of dark theme before the light theme is loaded.

**Comment by dylanh724** [comment]
Sadly, I experience a similar issue with sphinx_rtd_ theme, too! It's probably related:

https://github.com/readthedocs/readthedocs.org/issues/11528

**Comment by vict0rsch** [comment]
I do see this too, though I'm not sure how to fix this

**Comment by glassmancody** [comment]
This is definitely one of my favorite themes I've found, and I'm attempting to transition OpenMW's current documentation to use this.

I noticed this flashing when navigating through the TOC as well. Is there still not a known cause for this? I will try and set up a local build to try and figure out what exactly is issue over the weekend.
